### PR TITLE
fix the timestamp hack

### PR DESCRIPTION
**Pull request by escapewindow**
- add timestamp logging
- fix local signingscript testing

Other than the number of args mismatch, the main issue where we weren't running the hack was:
- env vars are strings
- we were probably comparing against a bool, so it was always false

### CODE REVIEW

**Comment by gbrownmozilla** [comment]
lgtm. 

With some of the other recent timestamp work I've also wanted to see info like timestamp_url and timestamp_style, but that's not urgent.

**Comment by escapewindow** [comment]
> lgtm.
> 
> With some of the other recent timestamp work I've also wanted to see info like timestamp_url and timestamp_style, but that's not urgent.

How's [fb40a2d](https://github.com/mozilla-releng/scriptworker-scripts/pull/508/commits/fb40a2dae80f8295cd2be562d9f95386a62b437c)?

**Comment by gbrownmozilla** [comment]
> > lgtm.
> > With some of the other recent timestamp work I've also wanted to see info like timestamp_url and timestamp_style, but that's not urgent.
> 
> How's [fb40a2d](https://github.com/mozilla-releng/scriptworker-scripts/pull/508/commits/fb40a2dae80f8295cd2be562d9f95386a62b437c)?

Awesome. Thanks.

**Comment by escapewindow** [comment]
https://github.com/mozilla-releng/scriptworker-scripts/pull/508/commits/c0ec368d83ee42474c79d5fb21d09e232358b233 is already pushed to production-signingscript, because production testing is the best testing. It looks like we have a plausible full installer for QA.

We can either review+merge this and add gecko when we're ready, or we can add gecko to this PR before merging (once we have `go`).

File: signingscript/src/signingscript/sign.py
Comment (gbrownmozilla) [inline signingscript/src/signingscript/sign.py]: Is tmpdir still needed for anything?

**Review by gbrownmozilla — APPROVED** [review]

**Comment by escapewindow** [comment]
> [c0ec368](https://github.com/mozilla-releng/scriptworker-scripts/commit/c0ec368d83ee42474c79d5fb21d09e232358b233) is already pushed to production-signingscript, because production testing is the best testing. It looks like we have a plausible full installer for QA.
> 
> We can either review+merge this and add gecko when we're ready, or we can add gecko to this PR before merging (once we have `go`).

I'll go with the former since g.brown reviewed :)

**Pull request merged**
